macros/probing/klicky_probe_overides.cfg

```diff
@@ -72,42 +72,39 @@ gcode:
     DOCK_PROBE
 
     # Restore nozzle location again at the end
-    _EXIT_POINT FUNCTION=PROBE_ACCURACY MOVE=1 MOVE_SPEED={St}
+    _EXIT_POINT FUNCTION=PROBE_ACCURACY MOVE=1 MOVE_SPEED={travel_speed}
 
 
 [gcode_macro QUAD_GANTRY_LEVEL]
 rename_existing: _BASE_QUAD_GANTRY_LEVEL
 description: Conform a moving, twistable gantry to the shape of a stationary bed with klicky automount
 gcode:
-    {% set V = printer["gcode_macro _USER_VARIABLES"].verbose %}
-    {% if V %}
+    {% set verbose = printer["gcode_macro _USER_VARIABLES"].verbose %}
+    {% if verbose %}
         { action_respond_info("Quad gantry leveling") }
     {% endif %}
-
+    _CG28
+    
     _CHECK_PROBE action=query
     ATTACH_PROBE
 
-    _BASE_QUAD_GANTRY_LEVEL {% for p in params
-            %}{'%s=%s ' % (p, params[p])}{%
-            endfor %}
+    _BASE_QUAD_GANTRY_LEVEL samples=1 {% for p in params %}{'%s=%s ' % (p, params[p])}{% endfor %}
     DOCK_PROBE
 
-
+ 
 [gcode_macro BED_MESH_CALIBRATE]
 rename_existing: _BASE_BED_MESH_CALIBRATE
 description: Perform Mesh Bed Leveling with klicky automount
 gcode:
 
-    {% set V = printer["gcode_macro _USER_VARIABLES"].verbose %}
-    {% if V %}
+    {% set verbose = printer["gcode_macro _USER_VARIABLES"].verbose %}
+    {% if verbose %}
         { action_respond_info("Bed Mesh Calibrate") }
     {% endif %}
 
     _CHECK_PROBE action=query
     ATTACH_PROBE
 
-    _BASE_BED_MESH_CALIBRATE {% for p in params
-           %}{'%s=%s ' % (p, params[p])}{%
-          endfor %}
+    _BASE_BED_MESH_CALIBRATE  samples=1 {% for p in params %}{'%s=%s ' % (p, params[p])}{% endfor %}

```

Comment:
by default I use 1 sample for bedmesh and QGL, but it may be better to put it in _USER_VARIABLES

Reply:
Ok, I see. But why not set 1 sample directly in the [probe] section then ?
I think, if it exist in a general klipper section, it's better to use it from there to avoid confusion